test/javascripts/lib/pretty-text-test.js.es6

```diff
@@ -270,6 +269,16 @@ QUnit.test("Quotes", assert => {
 </aside>`,
          "handles nested quotes properly");
 
+  assert.cookedOptions(`[quote="bob, post:1, topic:1"]\ntest quote\n[/quote]`, { lookupPrimaryUserGroupByPostNumber: function() { return "aUserGroup" } },
```

Comment:
I would put your args on a different line, and use ES6 function syntax:

`{ lookupPrimaryUserGroupByPostNumber: () => "aUserGroup" }`